pkg/daemon/ceph/client/filesystem.go

```diff
@@ -28,12 +27,6 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 )
 
-const (
-	// MultiFsEnv defines the name of the Rook environment variable which controls if Rook is
-	// allowed to create multiple Ceph filesystems.
-	MultiFsEnv = "ROOK_ALLOW_MULTIPLE_FILESYSTEMS"
```

Comment:
Since multiple filesystems is only allowed by default in Pacific, do we at least need release notes for nautilus/octopus for them to enable it if needed? Really they should update to Pacific if they want multiple filesystems though.

Reply:
How about also changing the commit description and PR title since this is a small breaking change?

Reply:
I've split each commit in a separate PR for more visibility https://github.com/rook/rook/pull/8359

Reply:
@leseb want to update the PR description?